iaso/api/completeness.py

```diff
@@ -19,12 +21,18 @@ class CompletenessViewSet(viewsets.ViewSet):
 
     def list(self, request):
         profile = request.user.iaso_profile
+        profile_org_units = OrgUnit.objects.filter(id__in=profile.org_units.all())
+
         queryset = (
             Instance.objects.filter(project__account=profile.account)
             .exclude(deleted=True)
             .exclude(file="")
             .with_status()
         )
+
+        if profile_org_units:
+            queryset = queryset.filter(org_unit__path__descendants=ArraySubquery(profile_org_units.values("path")))
```

Comment:
You can have something simpler if you use `filter_for_user()`:

```suggestion
            queryset = queryset.filter(org_unit__in=profile_org_units))
```